css/index.css

```diff
@@ -57,35 +60,80 @@ html, body {
 }
 
 h1, h2, h3, h4, h5 {
-    font-size: 18px;
+    font-size: 1.8rem;
     margin-bottom: 15px;
     font-family: 'Rubik', sans-serif;
 }
 
 p {
     line-height: 1.4;
+    font-size: 1.6rem;
 }
 
 .container {
-    width: 800px;
+    max-width: 800px;
+    width: 100%;
     margin: 0 auto;
 }
 
+header {
+    margin: 2% 0 5% 0;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+}
+
+header nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+header nav ul {
+    margin: 18px 5% 0; 
+    width: 90.6%;
+    justify-content: space-between;
+    display: flex;
+    font-size: 1.4rem;
+}
+
+header nav ul li a {
+    display: block;
+    text-decoration: none;
+    color: #000;
+}
+
+header .logo {
+    align-self: flex-end;
+    margin: 5px 1.5% 0 2%;
+}
+
 .top-content {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
-    margin-bottom: 20px;
     border-bottom: 1px dashed black;
+    margin: 10px 0 20px 2.3%;
```

Comment:
The `2.3%` margin-left here is giving your jumbotron image spacing on the left side in the 400px view. You could override that with your 400px media query.